packages/popover/README.md

```diff
@@ -7,25 +7,26 @@ npm i pcln-popover
 ```
 
 ```jsx
-import Popover from 'pcln-popover'
+// TODO: triage explicit import path
+import Popover from 'pcln-popover/dist/index.js'
```

Comment:
make sure this gets opened as an issue, not just a TODO in the codeline

Reply:
#504 has been created to follow up w. this `TODO` comment. (cc @sdalonzo )